### DISCUSSION TITLE
请问BASE_URL和PROXY_URL分别有什么用啊？

### DISCUSSION THREAD

**Discussion by morvenhuang** [ChatGPTNextWeb/NextChat#2335] [Q&A]
BASE_URL我大概理解，就是不直接访问OpenAI的API请求地址，而是通过某个地址中转。那PROXY_URL呢？

**Comment by InitialXKO** [comment]
PROXY_URL才是中转，使用OPENAI的KEY。BASE_URL是代替OPENAI，可能用一套其他的KEY来验证和计费
